aws/amazon-chime-sdk-component-library-react
# Create useMediaStreamMetrics hook to expose audio and video metrics

**Issue #:** 

**Description of changes:**
Add new hook called useMediaStreamMetrics to expose audio and video metrics.

**Testing Steps**
1. When user A joins a meeting, in the side bar, the user should able to see a signal strength icon. when user A clicks it, it should show a panel audio statistics and bandwidth statistics.
2. When the user A enables video, the panel should also include video statistics data
3. When there is another attendee ,user B, joins the same meeting, the user A will see a list of attendees in the roster. If user B  enables video, there will be a menu icon showing up in the roster cell of user B. When user A clicks it, it will show user B's video statistics as downstream stats. 
4. If user B disables video, the menu icon of user B in user A's roster list will hide. No stats of user B showing.
5. User A should also see their own video stats in the roster. The stats will be upstream stats.

**Testing**
1. Have you successfully run `npm run build:release` locally?
yes
2. How did you test these changes?
using meeting demo

3. If you made changes to the component library, have you provided corresponding documentation changes? yes

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
